classes/ai_manager_utils.php

```diff
@@ -526,7 +526,7 @@ public static function determine_purposes_availability(stdClass $user, int $cont
                 $purposes[] = [
                     'purpose' => $purpose,
                     'available' => self::AVAILABILITY_DISABLED,
-                    'errormessage' => get_string('error_purposenotconfigured', 'local_ai_manager'),
+                    'errormessage' => get_string('error_purposenotconfigured', 'local_ai_manager', $purpose),
```

Comment:
For a user faced message we need the pluginname fetched via get_string of the purpose here instead of the purpose identifier.